sync/config/triggers.yaml

```
@@ -35,5 +35,6 @@ tags:
   - cel_expressions.md: cel_expressions.md
   - create-ingress.yaml: create-ingress.yaml
   - create-webhook.yaml: create-webhook.yaml
+  - getting-started/README.md: getting-started/_index.md
```

There is an issue. The link is relative.
```html
 <a href="./getting-started" />
```

It causes the url to redirect to 
``/docs/triggers/install/getting-started/ ``
instead of 
``/docs/getting-started/_index.md``
This could be fixed by
```suggestion
  - getting-started/README.md: install/getting-started.md
```

---

Good catch.

This was working when I tested against my local copy of the triggers repo before the change was merged, I must have missed something in my PR.

We want this doc exposed at `/docs/triggers/getting-started/` as there's other content we'll also want to make available under this folder.

I'll take another look.